nodemon.json

```
@@ -0,0 +1,10 @@
+{
+  "watch": [
+    "src"
+  ],
+  "ext": "tsx,css,html,ts",
+  "ignore": [
+    "src/**/*.spec.ts"
+  ],
+  "exec": "vite build"
```

It would be nice to use the delay property to debounce the nodemon settings.

```suggestion
  "exec": "vite build",
  "delay": 300
```

---

https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/commit/3c86c8864ecfd7afe098d18c384a923fd2d43281